src/component.js

```diff
@@ -72,7 +72,8 @@ function doSearch(value, clearInput = false) {
         query = value;
     }
 
-    searchClient.search(query)
+    searchClient.search(query, algolia.indexes.settings)  //This is just test. Need to make the settings conditional
```

Comment:
This is the main relevant change

It just adds the settings specified in the search.
BUT this is not tested, and might not work for multi-setting case.

ON that note, I would propose this is changed so that there is ONLY the multi setting case and no options.
If you have just one setting then you do it as a single entry in the multi setting case.

On top of that the switcher should allow you to set that setting arbitrarily to control your facets based on the language tree, and do it by default.

On top of that, I get errors when the index is intialised. That's because the PUT is not supported by my API.
That's an error.

This is also v3 of API and probably need v4.

THIS IS JUST A TEST!